packages/editor/src/components/post-excerpt/panel.js

```diff
@@ -198,8 +198,7 @@ function PrivateExcerpt() {
 				ref={ setPopoverAnchor }
 				renderToggle={ ( { onToggle } ) => (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="274" alt="post-excerpt before" src="https://github.com/user-attachments/assets/d976ebb3-bae1-47e6-abe0-f3f31aa487d7">

</td>
    <td>
<img width="280" alt="Post excerpt after" src="https://github.com/user-attachments/assets/740cb821-6c7e-4cf3-99f6-c660efc3e5d6">

</td>
  </tr>
</table>


The style didn't change because there are explicit styles for `Add and excerpt` in code. But adding `__next40pxDefaultSize` will ensure that it will not be listed in lint next time